assets/scss/hero.scss

```diff
@@ -90,22 +90,74 @@
   }
 }
 
-.exhibition-page {
+.exhibition-page, .blog-post {
   background: white;
   margin-top: -3rem;
   padding-top: 0.25rem;
+  .title {
+    margin-bottom: 2rem;
+  }
   h1 {
+    font-size: 2.125rem;
     margin-top: 2.5rem;
   }
   p.lead {
     font-size: 1.5rem;
     color: $mediumgrey;
   }
+  p {
+    font-size: 1.125rem;
+  }
+  blockquote {
+    p {
+      font-size: 1.5rem;
+    }
+  }
   article, section p, section h1, h2:not(.is-size-1-5), h3, h4, h5, h6 {
     max-width: 45rem;
     margin-left: auto;
     margin-right: auto;
   }
+  article {
+    .author:last-of-type .comma {
+      display: none;
+    }
+  }
+  .article {
+    iframe {
+      border: none;
+      width: 100%;
+      min-height: 400px;
+      margin: 1.5rem auto
+    }
+    p {
+      line-height: 1.55;
+    }
+    h2, h3 {
+      font-weight: 600;
+    }
+    h3 {
+      font-size: $font-size-large;
+    }
+    button {
+      display: block;
+      margin: 2rem auto;
+      padding: 0.5rem 1rem;
+      color: $white;
+      font-size: $font-size-small;
+      text-transform: uppercase;
+      border-radius: $border-radius-small;
+      box-shadow: $boxshadow-light;
+      background-color: $blue;
+      border: none;
+    }
+    cite {
+      margin-bottom: 1.5rem;
+    }
+  }
+  .credits {
+    max-width: 720px;
+  }
```

Comment:
where is this used?

Reply:
.credits is used in the exhibitions.

good point about the authors, I keep doing everything purely the JS way and not going for the easier/better CSS solution, have changed. Still need to keep the comment spacing between {{ name }} and the closing </SmartLink> though because this was causing the underline text-decoration to continue after the author name in the space before organisation